README.md

```diff
@@ -1,5 +1,7 @@
 # Frost Guide
 
+# Moved to https://bitbucket.ciena.com/projects/BP_FROST/repos/frost-guide/browse . Keeping only for gh-pages branch
```

Comment:
I don't think we want to put the path to the other repo as it is internal.  I think just a note that the project was moved to internal site.